boot/freeldr/freeldr/arch/i386/pc/machpc.c

```diff
@@ -273,7 +273,7 @@ DetectPnpBios(PCONFIGURATION_COMPONENT_DATA SystemKey, ULONG *BusNumber)
 
             if (PnpBufferSize + DeviceNode->Size > PnpBufferSizeLimit)
             {
-                ERR("Buffer too small! Ignoring remaining device nodes.\n");
+                ERR("Buffer too small! Ignoring remaining device nodes. (i = %d)\n", i);
```

Comment:
```suggestion
                ERR("Buffer too small! Ignoring remaining device nodes. (NodeNumber: %d)\n", i);
```

Reply:
It looks like `i` and `NodeNumber` values may not be identical, so should not be mixed.
Do you want both?

Reply:
Ok I see, let's leave `i` only then.

Reply:
Can you explain to me why i and NodeNumber may not be identical?

Reply:
Passed by pointer in `PnpBiosGetDeviceNode` call, though I didn't looked into it.